### PR TITLE
add follow symbolic links options

### PR DESCRIPTION
**Pull request by PepeANZ**
Adds new option to follow or ignore symbolic links
This functionality is already provided by core actions glob function
Only exposing this and not all glob options to reduce complexity
This is useful when a project has many symbolic links, such as a monorepo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,11 @@ For assistance with breaking changes, see [MIGRATION.md](docs/MIGRATION.md).
     # Optional. Default is 'warn'
     if-no-files-found:
 
+    # If true, symlink directories will be used to search files.
+    # If false, symlink directories will be ignored to search files.
+    # Optional. Default is 'true'
+    followSymbolicLinks:
```
Comment (Johennes) [inline README.md:96]: Should this be `follow-symbolic-links`?

Comment (PepeANZ) [reply README.md]: Yes, it should be, good catch

File: src/shared/search.ts [hunk 2]
```diff
@@ -19,6 +19,15 @@ function getDefaultGlobOptions(): glob.GlobOptions {
   }
 }
 
+export function getGlobOptions(
+  followSymbolicLinks?: boolean
+): glob.GlobOptions {
+  return {
+    followSymbolicLinks,
+    ...getDefaultGlobOptions()
```
Comment (Johennes) [inline src/shared/search.ts:27]: I think the order is wrong here. `followSymbolicLinks: true` from `getDefaultGlobOptions` will always override the `followSymbolicLinks` property, won't it?

Comment (PepeANZ) [reply src/shared/search.ts:27]: You are correct

**Comment by sanjacob** [comment]
Why did you close this?

**Pull request closed without merging**